src/vm/arm64/crthelpers.asm

```diff
@@ -12,85 +12,13 @@
 
     TEXTAREA
 
-; Calls to JIT_MemSet is emitted by jit for initialization of large structs. 
-; We need to provide our own implementation of memset instead of using the ones in crt because crt implementation does not gurantee 
-; that aligned 8/4/2 - byte memory will be written atomically. This is required because members in a struct can be read atomically 
-; and their values should be written atomically.
-; 
```

Comment:
Remove misleading bogus comment.  there is no reason to believe these accesses need to be aligned on arm64 Windows when they are not aligned on other platforms